tests/Cookie/CookieJarTest.php

```diff
@@ -37,6 +37,11 @@ public function testCreatesFromArray()
         $this->assertCount(2, $jar);
     }
 
+    public function testEmptyJarIsCountable()
+    {
+        self::assertCount(0, new CookieJar());
```

Comment:
No, I wan't to actually invoke the `count` method. Because PHP 7.2 throws a warning when trying to count a non-countable things like `null`, which could be done internally (i.e counting internal property which was not initialized) . So I want to check that `CookieJar` does not throws the warning when calling count directly after constructing. 

https://wiki.php.net/rfc/counting_non_countables

The same thing I did with the `MockHandler`, but we have to fix it (see array default)